<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasWindow</class>
 <widget class="QDialog" name="CanvasWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1888</width>
    <height>967</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Canvas</string>
  </property>
  <widget class="QPushButton" name="resetCanvas">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>590</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset canvas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drawButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eraseButton">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QLabel" name="roomLabel">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>650</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Room:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startGameButton">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>590</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="timerLabel">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>630</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time left:</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerScore">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>620</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="chosenWord">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>660</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Chosen word:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageBox">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>510</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>510</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameChatLabel">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>60</y>
     <width>271</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
